frontend/src/test/store.jsx

```diff
@@ -4,7 +4,6 @@
  * Notably, this one doesn't have any logging, and supports an initialState.
  */
 
-import React from 'react';
```

Comment:
Please revert this change.

Reply:
yes, just figured that i mistakenly removed it, the test failed cos of this. I'll do that and revert the yarn.lock as well ASAP

Reply:
Excellent, thanks for your help!